scalding-commons/src/main/scala/com/twitter/scalding/commons/thrift/TBaseOrderedSerialization.scala

```diff
@@ -14,11 +14,11 @@ import scala.util.control.NonFatal
 
 import com.twitter.scalding.serialization.JavaStreamEnrichments._
 
-abstract class TBaseOrderedBinary[T <: TBase[_, _]] extends TProtocolOrderedSerialization[T] {
+abstract class TBaseOrderedSerialization[T <: TBase[_, _]] extends TProtocolOrderedSerialization[T] {
```

Comment:
is there any reason we need a macro on this one at all? What part is macro generated? Can't we just use runtime reflection at instantiation to get the minField?